CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# How to contribute
+
+MariaDB Server has a vibrant community contributing in a wide range of areas. There are many valuable ways you can contribute to MariaDB.
+
+### Engage online with other community members
+---
+-   [MariaDB on Zulip](https://mariadb.zulipchat.com/)
+-   [maria-developers mailing list](http://launchpad.net/~maria-developers)
+-   [maria-discuss mailing list](http://launchpad.net/~maria-discuss)
+-   [maria-docs mailing list](http://launchpad.net/~maria-docs)
+-   #maria IRC channel on freenode ([see the IRC page on the Knowledge Base](https://mariadb.com/kb/en/meta/irc-chat-servers-and-zulip-instance/)  for help with IRC).
+-   The MariaDB Foundation and MariaDB Corporation have a presence on Reddit, Twitter, Facebook and Google Plus. See the  [social media page](https://mariadb.com/kb/en/mariadb/social-media/).
+
+### Help document MariaDB
+----
+-   Contribute towards  [documenting MariaDB Server](https://mariadb.com/kb/en/meta/writing-editing-library-articles/)  and its ecosystem by adding new content or improving existing content.
+-   [Translate](https://mariadb.com/kb/en/meta/translating-library-articles/)  existing documentation.
+### Help debug and develop MariaDB
+-----
+-   [Report bugs](https://jira.mariadb.org/)
+-   Test development versions
+-   Triage and debug bugs
```

Comment:
Test - is duplicated by quality control. "Triage and debug bugs" is a bit vague.